chapter_preface/index_vn.md

```diff
@@ -264,7 +298,22 @@ Once we have taught you how some component works,
 we can just use the Gluon version in subsequent tutorials.
 -->
 
-*dịch đoạn phía trên*
+Xuyên cuốn sách, chúng ta sẽ làm việc với thư viện MXNet,
+một thư viện với một đặc tính hiếm có, đó là vừa
+đủ linh hoạt để nghiên cứu và đủ nhanh để tạo ra sản phẩm.
+Cuốn sách này sẽ dạy về khái niệm học sâu từ đầu.
+Thỉnh thoảng, chúng tôi sẽ muốn đào sâu vào những chi tiết chuẩn
```

Comment:
"fine details"  nên dịch toàn câu như sau

Thỉnh thoảng, chúng tôi muốn đào sâu vào chi tiết với một số mô hình (mạng neuron) mà thường bị ẩn đằng sau các hàm API Gluon của thư viện MXNet" (Gluon cung cấp các hàm API có giúp người dùng dễ thao tác với một mô hình mà không cần đi sâu về mô hình đó).